src/components/pages/events/filters/dropdown-with-two-select/dropdown-with-two-select.jsx

```diff
@@ -0,0 +1,137 @@
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import React, { useRef, useState } from 'react';
+
+import Checkbox from 'components/shared/checkbox';
+import useClickOutside from 'hooks/use-click-outside';
+import ChevronIcon from 'icons/chevron.inline.svg';
+
+const DropdownWithTwoSelect = ({
```

Comment:
The naming is a bit confusing to me, can we rename it to dropdown-with-two-levels?